index.bs

```diff
@@ -453,6 +450,69 @@ function poseToMatrix (pose) {
 </pre>
 </div>
 
+## VRFrameData ## {#interface-vrframedata}
+
+The VRFrameData interface represents all the information needed to render a single frame of a VR scene.
+
+<pre class="idl">
+[Constructor]
+interface VRFrameData {
+  readonly attribute DOMHighResTimeStamp timestamp;
+
+  readonly attribute Float32Array leftProjectionMatrix;
+  readonly attribute Float32Array leftViewMatrix;
+
+  readonly attribute Float32Array rightProjectionMatrix;
+  readonly attribute Float32Array rightViewMatrix;
+
+  readonly attribute VRPose pose;
+};
+</pre>
+
+### Attributes ### {#vrframedata-attributes}
+
+<dfn attribute for="VRFrameData">timestamp</dfn>
+Monotonically increasing value that allows the author to determine if position
+state data been updated from the hardware. Since values are monotonically
+increasing, they can be compared to determine the ordering of updates, as newer
+values will always be greater than or equal to older values. The timestamp
+starts at 0 the first time {{getCurrentFrameData()}} is invoked for a given
+{{VRDisplay}}.
+
+<dfn attribute for="VRFrameData">leftProjectionMatrix</dfn>
+A 4x4 matrix describing the projection to be used for the left eye's rendering,
+represented as a 16 element array in column major order. This value may be
+passed directly to WebGL's uniformMatrix4fv function. It is highly recommended
+that applications use this matrix without modification. Failure to use this
+projection matrix when rendering may cause the presented frame to be distorted
+or badly aligned, resulting in varying degrees of user discomfort.
+
+<dfn attribute for="VRFrameData">leftViewMatrix</dfn>
```

Comment:
Do we want to describe this as the inverse of the model matrix for the eye? Since to ray-cast each eye I need to know this to know that I should take the inverse in order to create a camera to pass to Raycaster. We accurately described sittingToStandingTransform which is great, but I expect this to be less used than Raycasting. I might use the pose for Raycasting, but I suspect there will be some interesting applications for the eye model matrix and getting back to it and knowing how and that I should seems worth calling out.

This is duplicated with rightViewMatrix right now so maybe its worth having a reference from each of these to the "conversions" section so that we don't have to type it twice (this includes the sittingtoStandingTransform.